src/overlays/actors/ovl_En_Syateki_Okuta/z_en_syateki_okuta.c

```diff
@@ -60,6 +60,15 @@ static ColliderCylinderInit sCylinderInit = {
     { 20, 40, -30, { 0, 0, 0 } },
 };
 
+typedef enum {
+    /* 0 */ SG_OCTO_ANIM_SHOOT, // unused
+    /* 1 */ SG_OCTO_ANIM_DIE,
+    /* 2 */ SG_OCTO_ANIM_HIDE,
+    /* 3 */ SG_OCTO_ANIM_FLOAT,
+    /* 4 */ SG_OCTO_ANIM_APPEAR,
+    /* 5 */ SG_OCTO_ANIM_HIT // unused
+} ShootingGalleryOctorokAnimation;
+
 static AnimationInfo sAnimationInfo[] = {
     { &gOctorokShootAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f },
```

Comment:
```suggestion
    /* 5 */ SG_OCTO_ANIM_HIT, // unused
    /* 6 */ SG_OCTO_ANIM_MAX
} ShootingGalleryOctorokAnimation;

static AnimationInfo sAnimationInfo[SG_OCTO_ANIM_MAX] = {
    { &gOctorokShootAnim, 1.0f, 0.0f, 0.0f, ANIMMODE_ONCE, -1.0f }, // SG_OCTO_ANIM_SHOOT
```

I've been adding `MAX` to the animations to make sure any added animations have their corresponding enum remain in sync.

Also possibly hasn't been finalised, but I like having enum values commented along with the animations (i.e. the `// SG_OCTO_ANIM_SHOOT`), but this does feel a little redundant once the values are named. It is a quick way to go from the enum to the Animation though. It becomes annoying though when the animation name is long enough to force a new line from the formatter.

Long story short, you don't need the comments by the side until we decide if we want them in my animation cleanup